MAINTAINERS

```diff
@@ -600,6 +616,11 @@ made through a pull request.
 	Email = "SvenDowideit@home.org.au"
 	GitHub = "SvenDowideit"
 
+	[people.thajeztah]
+	Name = "Sebastiaan van Stijn"
+	Email = "sebastiaan@gone.nl"
```

Comment:
I think github@gone.nl would be better, just to separate things.

(Might change it to `@dockr.nl` one day, which I registered for fun :smile:)